### ISSUE TITLE
Enchantments are removed when placed

### ISSUE THREAD

**Issue by Schinkennugget** [PssbleTrngle/CreateJetpack#140]
### What happened?

When you place the jetpack and then break it, it is no longer enchanted

### Loader

fabric

### Mod Version

4.2.1

### Minecraft Version

1.20

**Comment by Helionys** [comment]
i have the same issue the capacity enchant dissapear when break the jetpack placed in the floor filling the air but i use the forge ML version 4.3.2 in the 1.20.1 minecraft version

**Comment by Schinkennugget** [comment]
The problem only occured after updating Create and this mod, so maybe there is a compatibility issue with Create

**Comment by PssbleTrngle** [comment]
Is there a new create build? Which version are you on?

**Comment by Schinkennugget** [comment]
I’m on create-fabric-0.5.1-j-build.1609+mc1.20.1, but there’s also a newer build.1631

**Comment by Schinkennugget** [comment]
I’ve tested it out, also doesn’t work with the newest build

**Comment by PssbleTrngle** [comment]
Yes, this is because of the new create version, I will have to release a new jetpack version that supports it

**Comment by PssbleTrngle** [comment]
I have released a new version with support for the new create build that should hopefully fix this issue, if it still persists feel free to reopen this
